src/Microsoft.OpenApi.Readers/Microsoft.OpenApi.Readers.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
+        <TargetFramework>net8.0</TargetFramework>
```

Comment:
this effectively means we're dropping support for netfx, and most of net core except for the latest.
I'd like to make sure we've evaluated the consequences for our customers with the PM before we make changes to the code for the libraries (changing hidi and tests are fine) ? (I don't want us to have to backtrack the work later on when people complain about too high of a target)

Reply:
I can revert the libs to continue supporting netstandard2.0 so that we don't drop support for customers using it and only upgrade hidi and the tests.

Reply:
I think the upgrade is great for all other projects besides OpenAPI and readers. For those we should keep netstandard2.0 if we're not expecting huge benefits from new APIs, or make sure we make a case for it, understand how much of the audience we'd be "loosing" and probably target something lower like net6.